hadoop-hdds/server-scm/dev-support/findbugsExcludeFile.xml

```diff
@@ -47,4 +47,8 @@
     <Class name="org.apache.hadoop.hdds.scm.server.TestSCMSecurityProtocolServer" />
     <Bug pattern="URF_UNREAD_PUBLIC_OR_PROTECTED_FIELD" />
   </Match>
+  <Match>
+    <Class name="org.apache.hadoop.hdds.scm.metadata.TestSCMTransactionInfoCodec"/>
```

Comment:
This is the same handling way of `org.apache.hadoop.hdds.scm.metadata.TestOMTransactionInfoCodec`. Just keep consistent here.